meilisearch-release-changelogs/v0.28.0.md

```diff
@@ -0,0 +1,174 @@
+- The endpoint to update an index is now `PATCH /indexes/{indexUid}` instead of `PUT`
+- The already deprecated `name` field returned in the response body is totally removed.
+
+Done by @irevoire and @Kerollmops in #2423, #2452, and #2454.
+
+### Changes in `indexes/{uid}/documents` endpoints
+
+- When getting the documents via the `GET /indexes/{uid}/documents` route, the pagination has been improved. It means:
+  - You can still use the `offset` and `limit` parameters to browse the indexes.
+  - The documents are returned in the `results` field in the response body.
+  - The `limit`, `offset`, and `total` are also now available in the response body.
+- When getting the documents via the `GET /indexes/{uid}/documents` route, the `attributesToRetrieve` parameter has been renamed into `fields`
+- The `fields` parameter is available for the `GET /indexes/{uid}/documents/{document_uid}` route.
+- The [`displayedAttributes` settings](https://docs.meilisearch.com/reference/api/displayed_attributes.html) does not impact the `indexes/{uid}/documents` anymore, but ony the `/indexes/{uid}/search` endpoints.
+
+Done by @irevoire in #2372, #2433, and #2454.
+
+### Changes in `/tasks` and `/indexes/{uid}/tasks` endpoints
+
+- Add filtering on the `GET /tasks` endpoint:
+  - Add filtering capabilities on `type`, `status` and `indexUid` for `GET` `task` lists endpoints.
+  - It is possible to specify several values for a filter using the `,` character. e.g. `?status=enqueued,processing`
+  - Between two different filters, an `AND` operation is applied. e.g. `?status=enqueued&type=indexCreation` is equivalent to `status=enqueued AND type = indexCreation`
+- Add keyset pagination on the `GET /tasks` endpoints, allowing users to browse multiple sets of `task` items.
+  - You can use the `limit`, `from` query parameters to browse the tasks.
+  - The tasks are still returned in the `result` field in the response body.
+  - The `limit`, `from`, and `next` are available in the response body.
+- Remove `GET /indexes/:indexUid/tasks` is removed. It can be replaced by `GET /tasks?indexUid=:indexUid`
+- Remove `GET /indexes/:indexUid/tasks/:taskUid` is removed.
+- Rename `uid` to `taskUid` in the response body returned by every asynchronous task (ex: index creation, document addition...)
+- Rename some task properties
+  - `documentPartial`-> `documentAdditionOrUpdate`
+  - `documentAddition`-> `documentAdditionOrUpdate`
+  - `clearAll` -> `documentDeletion`
+
+Done by Kerollmops in #2399 and #2445.
+
+### Changes regarding the dumps
+
+- Dump creation becomes an asynchronous task. It means the dump task is put at the end of the task queue but will be processed in priority.
+- `POST /dumps` now returns `202` like any other asynchronous operation (e.g. index creation, document addition...
+- The `GET /dumps/:dump_uid/status` route is removed because not useful anymore, use `GET /tasks` instead.
```

Comment:
What is the `type` for a dump task? So that we can add a filter on `type` here. I made my best guess

```suggestion
- The `GET /dumps/:dump_uid/status` endpoint has been removed. It can be replaced by `GET /tasks?type=dump`.
```